modin/pandas/test/test_series.py

```diff
@@ -2136,10 +2150,14 @@ def test_lt(data):
 @pytest.mark.parametrize("level", [0, -1, None])
 def test_mad(level, data, axis, skipna):
     modin_series, pandas_series = create_test_series(data)
-    df_equals(
-        modin_series.mad(axis=axis, skipna=skipna, level=level),
-        pandas_series.mad(axis=axis, skipna=skipna, level=level),
-    )
+    try:
+        pandas_result = pandas_series.mad(axis=axis, skipna=skipna, level=level)
+    except Exception as e:
+        with pytest.raises(type(e)):
+            modin_series.mad(axis=axis, skipna=skipna, level=level)
+    else:
+        modin_result = modin_series.mad(axis=axis, skipna=skipna, level=level)
+        df_equals(modin_result, pandas_result)
```

Comment:
```suggestion
    eval_general(*create_test_series(data), lambda df: df.mad(axis=axis, skipna=skipna, level=level))
```
The same comments as for `test_all` func

Reply:
Fixed.